website/gatsby-config.js

```diff
@@ -83,8 +83,54 @@ module.exports = {
         ignore: ['.js'],
       },
     },
+
+    /**
+     * Transformers for making content available in graphql queries
+     */
+
+    // For querying images
     `gatsby-transformer-sharp`,
+
+    // For querying MDX
+    {
+      resolve: 'gatsby-mdx',
+      options: {
+        extensions: [
+          '.mdx',
+          // ".md"
+        ],
+        remarkPlugins: [require('remark-slug'), require('remark-emoji')],
+        gatsbyRemarkPlugins: [
```

Comment:
Those plugins are newly added, which are necessary for rendering optimized images from markdown pages